act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
+  </p></blockquote>
+</div>
+
+While an accessibility support section MUST be included in the ACT Rule, it MAY be empty when there are no known accessibility support issues.
  
 <div class=note>
   **Note:** WCAG Evaluation Methodology (WCAG-EM) provides guidance on defining an [accessibility support baseline](https://www.w3.org/TR/WCAG-EM/#step1c) for test scenarios, which can help users of ACT Rules to select the appropriate rules for their own circumstance.
 </div>
 
 
 Test Cases {#test-cases}
-========================
+------------------------
+
+Test cases are (snippets of) content that can be used to validate the implementation of ACT Rules. Each rule MUST have one or more test cases for `passed`, `failed`, and `inapplicable` [=outcome=]. A test case consists of two pieces of data, a snippet of each [input aspect](#input-aspects) for a rule, and the expected outcome for that rule. Test cases serve two functions, firstly as example scenarios for readers to understand when a rule passes, when it fails, and when it is inapplicable. It also serves developers and users of accessibility testing tools and methodologies to validate that a rule is correctly implemented.
 
-Test cases are (snippets of) content that can be used to validate the implementation of ACT Rules. Each rule MUST have one or more test cases for `passed`, `failed`, and `inapplicable` [=outcome=]. A test case consists of two pieces of data, a snippet of each [test aspect](#input-aspects) for a rule, and the expected result from evaluating that rule. Test cases serve two functions, firstly as example scenarios for readers to understand when a rule passes, when it fails, and when it is inapplicable. It also serves developers and users of accessibility testing tools and methodologies to validate that a rule is correctly implemented.
+<div class="example">
+  <p>**Example:** HTML test cases for a rule that checks if `img` elements have a text alternative.</p>
+  <blockquote>
+    <p>Example of a `passed` outcome:</p>
+    ```html
+<img alt="W3C Logo" src="image/w3c.png">
+    ```
+    <p>Example of a `failed` outcome:</p>
+    ```html
+<img src="image/w3c.png">
+    ```
+    <p>Example of an `inapplicable` outcome:</p>
+    ```html
+<input type="image" alt="W3C Logo" src="image/w3c.png">
+    ```
+  </blockquote>
+</div>
 
 
 Change Log {#change-log}
-========================
+------------------------
 
 It is important to keep track of changes to the ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All changes to an ACT Rule that can change the [=outcome=] of a evaluation MUST be recorded in a change log. The change log can either be part of the rule document itself or be referenced from it.
```

Comment:
**Editorial**: 

- "change the [=outcome=]" -> "change [=outcomes=]"
- "a evaluation" -> "an evaluation"